store-and-retrieve-data/index.md

```diff
@@ -5,55 +5,63 @@ description: Learn about the complete data storage and retrieval process in Data
 
 # Store and Retrieve Data Overview
 
-DataHaven is a verifiable storage network that separates storage from verification. Storage providers hold your file bytes off-chain, while the chain records compact on-chain commitments (“receipts”) so integrity can be checked at read time.
+DataHaven is a verifiable storage network that separates storage from verification. Storage providers hold your file bytes off-chain, while the chain records compact on-chain commitments. These commitments serve as receipts so data can be checked at read time.
 
-This section provides a high-level overview of how to create a bucket, upload, and retrieve files.
+This section provides a high-level overview of how to create a bucket, upload files, and retrieve them.
 
 ## How File Storage Works at a Glance
 
-1. **Pick an MSP and bucket**: Connect your wallet or app identity. Select a Main Storage Provider (MSP) and create or reuse a bucket. Today, access is owner-only; sharing/public access is planned. Set your desired replication factor (BSP replicas).
+1. **Pick an MSP and bucket**: Connect your wallet, select a Main Storage Provider (MSP), and create a bucket. Set your desired replication factor for backup copies. Currently, bucket access is owner-only.
```

Comment:
We don't need details like bucket access is owner-only here, that should be defined in the guide.
```suggestion
1. **Pick an MSP and bucket**: Connect your wallet, select a Main Storage Provider (MSP), and create a bucket. Set your desired replication factor for backup copies.
```

Reply:
It isn't mentioned in the Create a Bucket guide but, I also don't think we are clear as to exactly what the answer is on access. It seems like there will be shareable links to files and/or buckets but probably not any kind of passwords to protect them. So, I guess access is owner-only but, what does access mean? If I have the link and there is no password, I can "access" as far as read/see the files even if I can't change or delete them, yes? 

We need more info; if not, we should not mention this right now until we know more.

Reply:
cc: @aljosamakevic for visibility